libs/shared/services/storage/ir-schema.service.ts

```diff
@@ -78,6 +78,10 @@ export class IRSchemaService {
     return this.model?.isSubsectionValid(subSectionId) ?? false;
   }
 
+  canUploadFiles(subSectionId: string): boolean {
+    return this.model?.canUploadFiles(subSectionId) ?? false;
```

Comment:
it should load the model if it's not available

Reply:
I will refactor the service and do that work there. Thanks!